joomla/joomla-cms
# [5.1] CLI: Core Update - improve output

### Summary of Changes
The output of the CLI command to update the Joomla core currently is producing very bad output. The way the progressbar is used, all output is more or less deleted right away again and the progressbar is also not really very progressy...

This PR changes the output to keep all messages which were previously written on the screen and to actually display a progressbar.


### Testing Instructions
1. See how the core update on the CLI looks right now.
2. Apply the changes from this PR.
3. Change the version in `/libraries/src/Version.php to a lower version like 5.0.0.
4. Since your DB structure will be a mismatch to our perceived version and the update, we have to comment out the schema check on line 195 of `libraries/src/Console/UpdateCoreCommand.php` and add `$error = 0;` in the line below.
5. Run `php cli/joomla.php core:update`


### Actual result BEFORE applying this Pull Request
Output is really not helpfull.


### Expected result AFTER applying this Pull Request
Output is a lot more verbose.


### Link to documentations
Please select:
